specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -148,9 +175,8 @@ Given these elements, the number of test cases for each scenario can be calculat
 - Delivery mode: Normal, ExpectReplies
 
 **Alternate flows**
-
-1. Skill sends proactive message that updates the card.
-2. Skill sends proactive message that deletes the card.
+1. Skill sends proactive message that [updates the card](https://github.com/microsoft/BotBuilder-Samples/blob/22c01b317b9b7c8418eb1799a2031739637a8307/samples/csharp_dotnetcore/57.teams-conversation-bot/Bots/TeamsConversationBot.cs#L58). 
```

Comment:
I think we should combine these two into a new scenario: "A Teams skill sends update and delete activities"

Reply:
resolved